manifests/k/KDE/Dolphin/Nightly/master-b6ec4b1c9/KDE.Dolphin.Nightly.installer.yaml

```diff
@@ -2,15 +2,15 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: KDE.Dolphin.Nightly
-PackageVersion: master-b6ec4b1c9
+PackageVersion: master-271

```

Comment:
This needs to be the same as the version in the ARP entries, which in this case is `master-b6ec4b1c9`, so the original manifest is correct

Reply:
@karankad you beat me by mere seconds 😄